openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -63,6 +65,12 @@ def to_representation(self, user):
         :return: Dict serialized account
         """
         profile = user.profile
+        badges = settings.FEATURES.get('ENABLE_OPENBADGES') or False
+        badges = badges and BadgeAssertionSerializer(
```

Comment:
How large will this data get?  Should there be a parameter in the request that indicates whether or not to include the badges information?  A user's profile could be requested in many places.  For example, when get a list of profiles for all users in a discussion thread, we don't necessarily want all the details of each user.